apps/src/templates/sectionAssessments/MultipleChoiceSurveyQuestionDialog.jsx

```diff
@@ -0,0 +1,99 @@
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import Button from '@cdo/apps/templates/Button';
+import BaseDialog from '@cdo/apps/templates/BaseDialog';
+import i18n from '@cdo/locale';
+import DialogFooter from '@cdo/apps/templates/teacherDashboard/DialogFooter';
+import color from '@cdo/apps/util/color';
+import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
+import {multipleChoiceDataPropType} from './assessmentDataShapes';
+
+const styles = {
+  dialog: {
+    paddingLeft: 20,
+    paddingRight: 20,
+    paddingBottom: 20
+  },
+  instructions: {
+    marginTop: 20
+  },
+  answers: {
+    float: 'left',
+    width: 550
+  },
+  icon: {
+    color: color.level_perfect
+  },
+  iconSpace: {
+    width: 40,
+    float: 'left'
+  },
+  answerBlock: {
+    width: '100%'
+  },
+  answerLetter: {
+    width: 30,
+    float: 'left',
+    fontWeight: 'bold'
+  }
+};
+
+class MultipleChoiceSurveyQuestionDialog extends Component {
+  static propTypes = {
+    isDialogOpen: PropTypes.bool.isRequired,
+    closeDialog: PropTypes.func.isRequired,
+    questionData: multipleChoiceDataPropType.isRequired
+  };
+
+  render() {
+    const {questionData} = this.props;
+
+    // Questions are in markdown format and should not display as plain text in the dialog.
+
+    return (
+      <BaseDialog
+        useUpdatedStyles
+        isOpen={this.props.isDialogOpen}
+        style={styles.dialog}
+        handleClose={this.props.closeDialog}
+      >
+        <div>
+          <h2>{i18n.questionDetails()}</h2>
+          <div style={styles.instructions}>
+            <SafeMarkdown markdown={questionData.question} />
+          </div>
+          {questionData.answers && questionData.answers.length > 0 && (
```

Comment:
Worth it to add a test for this?

Reply:
probably! what specifically are you referring to? or just adding some testing for this component in general?

Reply:
General testing would be great! Specifically, however, I was referring to ensuring the correct number of answers render (or don't render).

Reply:
Seconding @jmkulwik

Reply:
👍